config/jobs/kubernetes/publishing-bot/publishing-bot-periodics.yml

```diff
@@ -0,0 +1,87 @@
+periodics:
+- name: periodic-publishing-bot
+  cluster: k8s-infra-prow-build-trusted
+  annotations:
+    testgrid-dashboards: sig-release-publishing-bot
+    testgrid-tab-name: periodic
+  spec:
+    initContainers:
+    - name: initialize-repos
+      command:
+      - /init-repo
+      - --alsologtostderr
+      - --config=/etc/munge-config/config
+      - --rules-file=/etc/publisher-rules/config
+      - 2>&1
+      image: gcr.io/k8s-staging-publishing-bot/k8s-publishing-bot
```

Comment:
Its a good practice to specify the image tag also.

Reply:
Using the `latest` image tag here.